### PR TITLE
Add PR Preview support

### PR DESCRIPTION
**Pull request by npm1**
Fixes https://github.com/WICG/event-timing/issues/42

### CODE REVIEW

File: .pr-preview.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "src_file": "index.bs",
+    "type": "bikeshed",
+    "params": {
```
Comment (domenic) [inline .pr-preview.json]: Hmm, I don't think you'll need all this for WICG specs :). You can probably just do something like https://github.com/WICG/import-maps/blob/master/.pr-preview.json which will give you a default appearance like https://pr-preview.s3.amazonaws.com/WICG/import-maps/pull/130.html

**Comment by domenic** [comment]
Oh wait, there's a trailing comma, which might break things.

File: .pr-preview.json (hunk 1 above)
Comment (npm1) [reply .pr-preview.json]: > Oh wait, there's a trailing comma, which might break things.

Removed

**Comment by npm1** [comment]
> Oh wait, there's a trailing comma, which might break things. (Edit: fixed while I was typing, apparently.)

Yep. I wish that didnt' break things though! Merging

**Pull request merged**